spec/middleman-prismic/preview_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+require "middleman-prismic/preview"
+
+describe Middleman::Prismic::Preview do
+  it "passes through requests outside of /preview" do
+    app = rack_app
+    allow(Kernel).to receive(:system)
+    env = stub_env("http://example.com/")
+
+    middleware = Middleman::Prismic::Preview.new(app)
+    code, env, message = middleware.call env
+
+    expect(Kernel).not_to have_received(:system)
+    expect(code).to eq 200
+    expect(message).to respond_to(:each)
+  end
+
+  context "given a request to /preview" do
+    it "intercepts the request" do
+      app = rack_app
+      allow(Kernel).to receive(:system)
+      env = stub_env("http://example.com/preview")
+
+      middleware = Middleman::Prismic::Preview.new(app)
+      code, env, message = middleware.call env
+
+      expect(code).to eq 302
```

Comment:
I'm not sure what to expect here for Kernel. Will it have received `:system`?

Reply:
It will have, but I don't care about it in this spec. I've put the `allow(Kernel).to receive(:system)` here just to stub it out so that the command is not actually called.